gpiozero/boards.py

```diff
@@ -1686,3 +1687,52 @@ def off(self):
         Turns the socket off.
         """
         self.value = False
+
+
+class JamHat(CompositeOutputDevice):
+    """
+    Extends :class: `CompositeOutputDevice` for the `ModMyPi JamHat` board.
+
+    There are 6 LEDs, two buttons and a tonal buzzer. The pins are fixed.
+
+    Usage:
+        from gpiozero import JamHat
+
+        hat = JamHat()
+        hat.lights_1.on()
+        hat.buzzer.play(2000)
+        hat.off()
+
+    :param bool pwm:
+    If ``True``, construct :class: PWMLED instances to represent each LED 
+    on the board. If ``False`` (the default), construct regular :class: 
+    LED instances.
+
+    :param bool initial_value:
+    If ``False`` (the default), all LEDs will be off initially. If
+    ``None``, each device will be left in whatever state the pin is found
+    in when configured for output (warning: this can be on). If ``True``,
+    the device will be switched on initially.
+
+    :param Factory pin_factory:
+    See :doc:`api_pins` for more information (this is an advanced feature
+    which most users can ignore).
+
+    .. _ModMyPi JamHat: https://www.modmypi.com/jam-hat
+
+    """
+    def __init__(self, pwm=False, initial_value=False, pin_factory=None):
```

Comment:
Perhaps the `__init__` for JamHat should also have an option for the frequency to get passed into the buzzer's init?

Reply:
I think that's complicated, and it's easy enough to set immediately after.

Reply:
...unless you want to turn on the buzzer initially to a specific frequency (i.e. `initial_value=True`) without having an audible glitch :wink: